README.md

```diff
@@ -232,20 +232,21 @@ Not all options are turned on by default. Refer to `Tomb1Main.json5` for details
 - added save game crystals game mode (enabled via gameflow)
 - added pause screen
 - added movable camera on W,A,S,D
-- added Xbox One Controller support
-    - Per Axis Dead Zone
-    - Left Stick = movement
-    - A = Jump/Select
-    - B = Roll/Deselect
-    - X = Action/Select
-    - Y = Look/Select
-    - LB = Walk
-    - RB = Draw Weapons
-    - Dpad Up = Draw Weapons
-    - Back = Option
-    - Start = Pause
-    - Right Stick = Camera Movement
-    - R3 = Reset Camera
+- added controller support
+    | Xbox button | DualShock button   | Action          |
+    | ----------- | ------------------ | --------------- |
+    | Left Stick  | Left stick         | Movement        |
+    | A           | X                  | Jump/select     |
+    | B           | Circle             | Roll/deselect   |
+    | X           | Square             | Action/select   |
+    | Y           | Triangle           | Look/select     |
+    | RB          | R1                 | Walk            |
+    | LB          | L1                 | Draw weapons    |
+    | D-pad       | D-pad              | Movement        |
+    | Back        | Share/Touchpad     | Inventory       |
+    | Start       | Options            | Pause           |
+    | Right Stick | Right stick        | Camera movement |
+    | R3          | R3                 | Reset camera    |
```

Comment:
Should we match the PS1 controller layout? (I didn't update the Xbox column)
```
    | Xbox button | DualShock button   | Action          |
    | ----------- | ------------------ | --------------- |
    | Left Stick  | Left stick         | Movement        |
    | A           | Square                  | Jump/select     |
    | B           | Circle             | Roll/deselect   |
    | X           | X             | Action/select   |
    | Y           | L1           | Look/select     |
    | RB          | R1                 | Walk            |
    | LB          | Triangle                 | Draw weapons    |
    | D-pad       | D-pad              | Movement        |
    | Back        | Share/Touchpad     | Inventory       |
    | Start       | Options            | Pause           |
    | Right Stick | Right stick        | Camera movement |
    | R3          | R3                 | Reset camera    |
```
This could also be helped with #564 to allow proper custom control schemes.

Reply:
Can you also provide which buttons act like select and which deselect?

Reply:
Should be in the table. But after checking out #545 I did a little digging.

TLDR: The select / deselect button has changed a bunch depending on console, region, and game. Since the PS5 release, X is select and circle is deselect in all regions. I vote we go with that. Custom controls potentially in the future will allow that to be changed.

Long story:
X was always select (for non Asian regions). However, for deselect, it seems different buttons were used. Back in the PS1 days, I remember triangle being the primary deselect button for games. For TR1 though, they used square as deselect for some reason. TR2+ changed deselect to be triangle which matched most games for the PS1-PS2 era.

Since the PS3, X has been select and circle has been deselect. In Asian regions this was reversed, and X was deselect and circle was select. Since the PS5, there is now an all region standard: X is select and circle is deselect.